File: problems/beginner/1008/1008.rs
use std::io;

fn read_line_from_stdin() -> String {
	let mut string = String::new();

	io::stdin()
		.read_line(&mut string)
		.expect("Error while reading from stdin");

	string
}

fn get_i32_from_stdin() -> i32 {
	let num: i32 = read_line_from_stdin()
		.trim()
		.parse()
		.expect("Error while converting String into i32");

	num
}

fn get_f64_from_stdin() -> f64 {
	let num: f64 = read_line_from_stdin()
		.trim()
		.parse()
		.expect("Error while converting String into f64");

	num
}

fn main () {
	let employee_id = get_i32_from_stdin();
	let hours_worked = get_i32_from_stdin();
	let hour_value = get_f64_from_stdin();
	
	println!("NUMBER = {}", employee_id);
	println!("SALARY = U$ {:.2}", (hours_worked as f64) * hour_value);
}

File: problems/beginner/1008/1008.dart
import 'dart:convert';
import 'dart:io';

String input() {
  return (stdin.readLineSync(encoding: utf8) as String);
}

double doubleInput() {
  return double.parse(input().trim());
}

int intInput() {
  return int.parse(input().trim());
}

void main() {
  var employee_id = intInput();
  var employee_worked_hours = intInput();
  var employee_hour_value = doubleInput();

  var salary = employee_worked_hours.toDouble() * employee_hour_value;

  print("NUMBER = ${employee_id}");
  print("SALARY = U\$ ${salary.toStringAsFixed(2)}");
}
